.gitignore

```diff
@@ -100,3 +100,4 @@ src/scaffolding.config
 
 # Visual Studio Code
 .vscode
+src/ServicePulse.Host/angular/package-lock.json
```

Comment:
Why are we ignoring this package lock and not the Vue one?

Reply:
package-lock should not be ignored, i assume there was an issue on the angular side?